invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/formatters.js

```diff
@@ -20,19 +20,29 @@ function formatLoanToTableView(loan, actions = null) {
   const startDate = loan.metadata.start_date
     ? toShortDate(fromISO(loan.metadata.start_date))
     : null;
-  const endDate = loan.metadata.start_date
+  const endDate = loan.metadata.end_date
     ? toShortDate(fromISO(loan.metadata.end_date))
     : null;
+  const requested = loan.metadata.request_start_date
+    ? fromISO(loan.metadata.request_start_date).toRelativeCalendar()
+    : null;
+  const expired = loan.metadata.end_date
```

Comment:
@topless thanks for fixing the `loan.metadata.end_date`. 

I am a bit lost with these `requested` and `expired`: they are already above. Why do you need it 2 times?

Reply:
I wanted to use a table header that sounds better with relative time something like `requested 4 days ago` instead of `start date 4 days ago`

@ntarocco this also got @kprzerwa's attention so it may worth revisiting. Before doing so though, please have a look at @zzacharo proposal #551

Reply:
@ntarocco actually its not even that big of the deal I will remove the duplicates on my upcoming pr.